modules/HomePage/components/TaskList/TaskList.tsx

```diff
@@ -85,14 +86,15 @@ const TaskList: React.FC<TaskListProps> = ({
     const updatedTask: Task = {
       ...task,
       title: editFormState.title,
-      summary: editFormState.description,
+      description: editFormState.description,
       dueDate: new Date(editFormState.dueDate),
       priority: editFormState.priority,
     };
     saveTask(updatedTask);
     setEditFormState({ title: "", description: "", dueDate: "", priority: 1 });
   };
-
+  const { data: todos } = useGetTodosQuery();
```

Comment:
I think we can also handle isLoading from the query and display a loading indicator based on its value.